org.freedesktop.LinuxAudio.Plugins.Calf.json

```
@@ -0,0 +1,51 @@
+{
+    "id": "org.freedesktop.LinuxAudio.Plugins.Calf",
+    "branch": "21.08",
+    "runtime": "org.freedesktop.LinuxAudio.BaseExtension",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk//21.08",
+    "build-extension": true,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/app/extensions/Plugins/Calf",
+        "prepend-ld-library-path": "/app/extensions/Plugins/Calf/lib",
+        "prepend-pkg-config-path": "/app/extensions/Plugins/Calf/lib/pkgconfig"
+    },
+    "cleanup": [
+        "/bin",
+        "/share/applications",
+        "/share/bash-completion",
+        "/share/doc",
+        "/share/icons",
+        "/share/man"
+    ],
+    "modules": [
+        "shared-modules/linux-audio/fluidsynth2.json",
```

```suggestion
        "shared-modules/linux-audio/fluidsynth2-static.json",
```

The static linking version should be used for plugins.